packages/lib-classifier/src/plugins/tasks/models/Annotation.js

```diff
@@ -34,8 +36,8 @@ const Annotation = types.model('Annotation', {
     update (value) {
       self.value = value
       self._inProgress = true
-      const { authClient } = getRoot(self)
-      authClient?.checkBearerToken()
```

Comment:
These lines are left over from when I was debugging expired auth sessions in Beyond Borders, a long time ago. I forgot to remove them when I was finished.